test/src/tests/generated/httpinfrastructure/models.go

```diff
@@ -70,18 +70,18 @@ type Error struct {
 }
 
 // Response returns the raw HTTP response object.
-func (e Error) Response() *http.Response {
-	return e.rawResponse
+func (eVar Error) Response() *http.Response {
```

Comment:
Curious why this var name changed?  Not a big deal, just wondering.

Reply:
because we added a couple of reserved names and `e` was one of them.